### PR TITLE
Remove UWP (uap) references and special cased code

### LINKED ISSUE #2482: [Feature] Remove uap/UWP references
See #2323 for discussion

### PR DESCRIPTION
**Pull request by ErikEJ**
Some boy scouting on DataTestUtility (removed unused code)

fixes #2482

### CODE REVIEW

**Comment by JRahnama** [comment]
@ErikEJ personally, I support this pull request, but @David-Engel should include the conclusion.

**Comment by ErikEJ** [comment]
@JRahnama Of course!

File: src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs [hunk 1]
```diff
@@ -330,8 +325,6 @@ public static bool GetSQLServerStatusOnTDS8(string connectionString)
             return isTDS8Supported;
         }
 
-        public static bool IsNotX86Architecture => RuntimeInformation.ProcessArchitecture != Architecture.X86;
```
Comment (JRahnama) [inline src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs:333]: this is being used in DistributedTransactionTest. Failure is due to removing this one.

**Review by David-Engel — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs [hunk 2]
```diff
@@ -998,47 +998,6 @@ internal enum FedAuthInfoId : byte
...
-                case SniContext.Snix_Close:
-                    return "Snix_Close";
-                case SniContext.Snix_SendRows:
-                    return "Snix_SendRows";
-                default:
-                    Debug.Fail($"Received unknown SniContext enum. Value: {sniContext}");
-                    return null;
-            }
-        }
-
```
Comment (JRahnama) [inline src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs:1041]: Can you remove reference to `System.Diagnostics` at the top of this file please?

**Comment by JRahnama** [comment]
LGTM, with just a minor adjustment. Thank you @ErikEJ .

**Review by JRahnama — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs (hunk 2 above)
Comment (ErikEJ) [reply src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsEnums.cs:1041]: Fixed usings

**Comment by JRahnama** [comment]
@ErikEJ if you can address the conflict this is ready to merge.

**Comment by ErikEJ** [comment]
@JRahnama Which conflict?

**Comment by JRahnama** [comment]
> @JRahnama Which conflict?

This is weird. I saw the conflict this morning. Ignore my comment

**Pull request merged**